packages/block-editor/src/components/skip-to-selected-block/index.js

```diff
@@ -19,17 +19,15 @@ const SkipToSelectedBlock = ( { selectedBlockClientId } ) => {
 		selectedBlockElement.focus();
 	};
 
-	return (
-		selectedBlockClientId && (
-			<Button
-				isSecondary
-				className="block-editor-skip-to-selected-block"
-				onClick={ onClick }
-			>
-				{ __( 'Skip to the selected block' ) }
-			</Button>
-		)
-	);
+	return selectedBlockClientId ? (
+		<Button
+			isSecondary
+			className="block-editor-skip-to-selected-block"
+			onClick={ onClick }
+		>
+			{ __( 'Skip to the selected block' ) }
+		</Button>
+	) : null;
```

Comment:
Spotted an error with this, the component could return `undefined` which causes React to throw an error. It only seems to be an issue in the navigation screen and noticeable with this change because the block couldn't be deselected before.